content/en/_index.md

```diff
@@ -26,25 +26,19 @@ IAM, Kubernetes, Elastic Load Balancing etc -- that's what Opta does!
   </iframe>
 </p>
 
-## Who is it for?
+## Why use Opta
+Infrastructure as code (IaC) has rapidly become the standard for provisioning and managing Infrastructure and 
+for the right reasons! But the leading IaC tools are still complicated to use and require deep Cloud/Infrastructure
+expertise. Opta was conceptualized to help address this complexity. Opta is a simpler IaC framework with best practices
+built-in. It enables users to set up automated, scalable and secure infrastructure without being a cloud expert or 
+spending days figuring out cloud minutiae!
```

Comment:
After reading this paragraph I understand your idea of having some graphics to convey our message. Something to consider

Reply:
yeah that's in my list of todos!